pkg/domain/entities/images.go

```diff
@@ -242,8 +242,9 @@ type ImageSearchReport = entitiesTypes.ImageSearchReport
 
 // Image List Options
 type ImageListOptions struct {
-	All    bool     `json:"all" schema:"all"`
-	Filter []string `json:"Filter,omitempty"`
+	All                bool     `json:"all" schema:"all"`
+	ExtendedAttributes bool     // no json needed
```

Comment:
a comment the you use it for the libpod endpoint to add extra data would be welcome.